static/app/views/explore/components/schemaHintsDrawer.tsx

```diff
@@ -31,7 +32,12 @@ function SchemaHintsDrawer({hints}: SchemaHintsDrawerProps) {
 
   const selectedFilterKeys = useMemo(() => {
     const filterQuery = new MutableSearch(exploreQuery);
-    return filterQuery.getFilterKeys();
+    const allKeys = filterQuery.getFilterKeys();
+    // When there is a filter with a negation, it stores the negation in the key.
+    // To ensure all the keys are represented correctly in the drawer, we must
+    // take these into account.
```

Comment:
oh this reminds me, we'll also need to remove "has" filters from the list too
![Screenshot 2025-03-24 at 12 27 37 PM](https://github.com/user-attachments/assets/e27bf9a3-4a08-41b3-910b-5371b287b6d4)